salt/orch/update.sls

```diff
@@ -60,6 +60,7 @@ update_modules:
 {{ master_id }}-wait-for-start:
   salt.wait_for_event:
     - name: salt/minion/*/start
+    - timeout: 1200
```

Comment:
What about moving it to a pillar? It would be easier to find if we need to change it in the future...